src/components/voting/VoterButtons.tsx

```diff
@@ -57,25 +59,23 @@ const VoterButton = ({
 
   const isActive = kind === reactionType
 
-  if (id === '482') {
-    console.log('kind', kind, kind === reactionType, isActive)
-  }
-
   const color = isUpvote ? '#00a500' : '#ff0000'
 
-  const isUpdate = kind !== reactionType
-  const changeReactionTx = isUpdate
-    ? 'reactions.updatePostReaction'
-    : 'reactions.deletePostReaction'
+  const isDelete = kind === reactionType
+  const changeReactionTx = isDelete
+    ? 'reactions.deletePostReaction'
+    : 'reactions.updatePostReaction'
 
-  const updateOrDelete = useCallback((update: boolean, _reactinoId?: ReactionId) => {
+  const updateOrDelete = (isDelete: boolean, _reactinoId?: ReactionId) => {
```

Comment:
Why do we need to pass `isDelete` as an additional param here? `isDelete` already exists as a const and we can use it?